package.json

```diff
@@ -219,23 +219,22 @@
   },
   "devDependencies": {
     "@artsy/antigravity": "0.2.0",
-    "@babel/cli": "7.10.5",
-    "@babel/core": "7.11.1",
-    "@babel/plugin-proposal-class-properties": "7.10.4",
-    "@babel/plugin-proposal-decorators": "7.10.5",
-    "@babel/plugin-proposal-nullish-coalescing-operator": "7.10.4",
-    "@babel/plugin-proposal-optional-chaining": "7.11.0",
-    "@babel/plugin-syntax-dynamic-import": "7.8.3",
-    "@babel/plugin-transform-modules-commonjs": "7.10.4",
-    "@babel/plugin-transform-runtime": "7.11.0",
-    "@babel/preset-env": "7.11.0",
-    "@babel/preset-react": "7.10.4",
-    "@babel/preset-typescript": "7.10.4",
-    "@babel/register": "7.10.5",
-    "@babel/runtime": "7.11.2",
+    "@babel/cli": "^7.12.1",
```

Comment:
Might be a good idea to extract these babel updates into their own PR if we can easily isolate.